### PR TITLE
Minor reflow/refactor

### PR DESCRIPTION
**Pull request by KOOKIIEStudios**
Reflow and refactored switch statement in MapleCharacter.checkForceShield()
Moved IDs from there to a HashMap in GameConstants
Minor translation of text in MapleMonster

### CODE REVIEW

File: AzureMS/src/client/Character/MapleCharacter.java [hunk 1]
```diff
@@ -5752,192 +5752,16 @@ public void checkForceShield() {
...
-                break;
-            default:
-                equip = null;
+
+        int equipID = GameConstants.checkSecondary.getOrDefault(job, 0);
+        if (equipID != 0){//check that the jobID has a corresponding secondary weapon coded
+            equip = (Equip) li.getEquipById(equipID);
+            potential = true;
         }
+        else{
```
Comment (Bratah123) [inline AzureMS/src/client/Character/MapleCharacter.java]: else {

File: AzureMS/src/server/LifeEntity/MobEntity/MonsterEntity/MapleMonster.java [hunk 2]
```diff
@@ -1196,7 +1196,7 @@ public final void handleSteal(MapleCharacter chr) {
                         idrop = new Item(d.itemId, (byte) 0,
                                 (short) (d.Maximum != 1 ? Randomizer.nextInt(d.Maximum - d.Minimum) + d.Minimum : 1),
                                 (byte) 0);
-                        idrop.setGMLog(chr.getName() + "가 스틸로 인한 아이템 훔치기로 얻은 아이템");
+                        idrop.setGMLog(chr.getName() + "obtained items using Steal.");
```
Comment (Bratah123) [inline AzureMS/src/server/LifeEntity/MobEntity/MonsterEntity/MapleMonster.java]: chr.getName() + " obtained items using Steal."

File: AzureMS/src/constants/GameConstants.java [hunk 3]
```diff
@@ -7960,5 +7956,50 @@ public static boolean isIllium(final int job) {
     public static boolean isMichael(final int job) {
         return job == 5000 || (job >= 5000 && job <= 5112);
     }
-    
+
+    //methods to initialise hashmap for MapleCharacter.checkForceShield()
+    //EXPERIMENTAL; lazy intialisation
+    // contents: <jobID, equipID>
+    public static final Map<Short, Integer> checkSecondary = new HashMap<Short, Integer>(){{
```
Comment (Bratah123) [inline AzureMS/src/constants/GameConstants.java]: CHECK_SECONDARY

File: AzureMS/src/server/LifeEntity/MobEntity/MonsterEntity/MapleMonster.java (hunk 2 above)
Comment (KOOKIIEStudios) [reply AzureMS/src/server/LifeEntity/MobEntity/MonsterEntity/MapleMonster.java]: hey. I just followed the original format, kay 🤙

File: AzureMS/src/constants/GameConstants.java (hunk 3 above)
Comment (KOOKIIEStudios) [reply AzureMS/src/constants/GameConstants.java]: I was just following the conventions in the constants for the server but sure 🤙

**Pull request merged**